azure-pipelines-official.yml

```diff
@@ -135,27 +140,49 @@ extends:
         displayName: Official Build
         timeoutInMinutes: 360
         templateContext:
+          # This ensures that the SDL analysis runs only once and not for each output separately.
+          # https://eng.ms/docs/cloud-ai-platform/devdiv/one-engineering-system-1es/1es-docs/1es-pipeline-templates/features/outputs#multiple-outputs
+          outputParentDirectory: '$(Build.SourcesDirectory)/artifacts'
```

Comment:
Technically this is orthogonal to fixing binskim (that could be done just by setting the `analyzeTargetGlob` parameter above), but without `outputParentDirectory`, the SDL analysis (including 10min of BinSkim analysis) would run for each output we upload separately.